src/main/java/dk/kb/labsapi/ImageExport.java

```diff
@@ -147,9 +155,62 @@ public QueryResponse illustrationSolrCall(String query, Integer startTime, Integ
         solrQuery.setHighlight(false);
         solrQuery.set(GroupParams.GROUP, false);
 
+        QueryResponse response = callSolr(solrQuery);
+        return response;
+    }
+
+    public QueryResponse fullpageSolrCall(String query, Integer startTime, Integer endTime, Integer max) throws IOException {
+        validateQueryParams(startTime, endTime, max);
+        int usableStartTime = setUsableStartTime(startTime);
+        int usableEndTime = setUsableEndTime(endTime);
+
+        // Construct solr query with filter
+        String filter = "recordBase:doms_aviser AND py:[" + usableStartTime + " TO "+ usableEndTime + "]";
+        log.info("The query gets filtered with the following filter: " + filter);
+        SolrQuery solrQuery = new SolrQuery();
+        solrQuery.setQuery(query);
+        solrQuery.setFilterQueries(filter);
+        solrQuery.setFields("pageUUID, page_width, page_height");
+        solrQuery.setRows(max == -1 ? defaultExport : max);
+        solrQuery.setFacet(false);
+        solrQuery.setHighlight(false);
+        solrQuery.set(GroupParams.GROUP, false);
+
+        QueryResponse response = callSolr(solrQuery);
+        return response;
```

Comment:
This is nearly identical to rhe `illustrationSolrCall`. Perhaps use a helper method for constructing a `SolrQuery` with the common parts, then extend with `filter` + ` `fields` in the `*SolrCall` methods?